### ISSUE TITLE
Credit card icons return error

### ISSUE THREAD

**Issue by melindahelt** [woocommerce/woocommerce-gateway-paypal-express-checkout#596]
2200569-zen

What I expected:
- I expected clickable items at checkout to work and not create an error.

What happened:
- When clicking on the credit card icons, as if to select payment method, an error happens. In reality, I believe those icons should not be clickable as the credit card type is auto identified when entering in the credit card number on the next screen.

See error with Storefront, WooCommerce 3.6.5 and PayPal Checkout 1.6.16

Confirmed that this happens on both cart and checkout pages.

Clicking the yellow PayPal button works as expected, but  clicking the credit  card icons result in an error.

![](https://cld.wthms.co/AXpiaF+)Image Link: https://cld.wthms.co/AXpiaF

**Comment by dechov** [comment]
Unable to reproduce:

![paypal-checkout-card-click](https://user-images.githubusercontent.com/1867547/62261170-66315280-b3e2-11e9-9247-e26046c27a33.gif)

Any idea as to what could be different between the setups? Also reported in https://wordpress.org/support/topic/error-when-clicking-the-credit-card-button/#post-11787704

**Comment by syllie** [comment]
Hi, I arrived here because we also experience issues with the credit card icons, though my situation may be slightly different. On click on any of the credit card images we get a non descriptive error message 
![image](https://user-images.githubusercontent.com/1240898/62262388-d92adf80-b45b-11e9-8c12-73f3a5be24a5.png).

Hitting the 'Try Again' button opens the paypal popup as expected (but how many customers leave instead). We have 4 commercial sites that are suddenly all affected. Only other information I can provide is that I am in Australia, and the console shows a "You have not loaded metadata for `"AU"` country" javascript error. I am trying to confirm if something similar happens to US customers too.

![image](https://user-images.githubusercontent.com/1240898/62262608-a7664880-b45c-11e9-92e4-ec4655da5631.png)

**Comment by dechov** [comment]
Thanks for reporting – I was able to reproduce the latter issue just by changing the customer country to anything other than US (on the Checkout page, or on the Cart page under Shipping).

Not sure why this would be happening, but I see that PayPal has recently made changes to the credit card form – having it render inline instead of in a modal, as shown in the screenshot – and the breakage may well be related, especially if the "Try again" button shows the modal and it works. Will keep looking into what the cause may be.

The 10002 error reported in the original issue may be unrelated (though possibly also a result of changes on PayPal's side to the way the credit card form is displayed).

**Comment by dechov** [comment]
PayPal has confirmed they are looking at an issue on their end related to country codes and credit card checkout – we're expecting an update next week and will continue to track both the country issue and the 10002 code reported in the original post.

**Comment by fhaps** [comment]
For the record, I was able to reproduce the same error (`Something went wrong. We'll take you back to checkout so you can try again`), even when I used the United States as the billing country (and the shop base address country).

See screencast here: https://cld.wthms.co/ZAqzU6

As mentioned before, clicking on `Try again` opens up the modal window, which works correctly.

2188081-zen

**Comment by card1concepts** [comment]
Our client is also experiencing this issue. And I went through the same thing where I could not reproduce the issue by simply clicking on the buttons as they would work fine as shown in the screenshot sequence above. However, the client pointed out to me that the error message appears for them only after entering credit card information and clicking the "PayNow" button. This occurs with a valid US address.

**Comment by card1concepts** [comment]
Has there been any progress finding a solution to this issue?

**Comment by laidlow73** [comment]
Have the same issue as **dechov** .

> 
> 
> Thanks for reporting – I was able to reproduce the latter issue just by changing the customer country to anything other than US (on the Checkout page, or on the Cart page under Shipping).
> 
> Not sure why this would be happening, but I see that PayPal has recently made changes to the credit card form – having it render inline instead of in a modal, as shown in the screenshot – and the breakage may well be related, especially if the "Try again" button shows the modal and it works. Will keep looking into what the cause may be.
> 
> The 10002 error reported in the original issue may be unrelated (though possibly also a result of changes on PayPal's side to the way the credit card form is displayed).

Any updates on this?

**Comment by LeRobert** [comment]
Same issue here. If I select PayPal for the payment, it opens the modal login window of PayPal correctly. But if I click on a credit card, it processes for a sec and says: "Something went wrong. We'll take you back to checkout so you can try again". If I then click on "Try Again", it opens the modal login window and works fine.

My locale is "SI".

I did some web traffic analysis and here are errors that are reported, if it helps. With some requests I also saw that although my locale is 'SI', there is 'US' sent as a country in communication with the PayPal API.


VM59 vendor.059a2357d9a678cb63bd91ee0b185f6c.js:1 
Error: You have not loaded metadata for `"SI"` country
    at s (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at r.value [as validateAddress] (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at Function.mapToProps (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at r (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at Function.r.mapToProps (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at r (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at s (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1
    at Object.run (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at d.c.initSelector (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)

VM54 checkout.js:2234 
ppxo_inline_guest_window_onerror {timestamp: 1566475803664, message: "Script error.", details: undefined, file: "", line: 0, …}

VM59 vendor.059a2357d9a678cb63bd91ee0b185f6c.js:1 
Uncaught Error: You have not loaded metadata for `"SI"` country
    at s (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at r.value [as validateAddress] (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at Function.mapToProps (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at r (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at Function.r.mapToProps (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at r (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at s (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1
    at Object.run (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)
    at d.c.initSelector (VM61 app.059a2357d9a678cb63bd91ee0b185f6c.js:1)

**Comment by card1concepts** [comment]
Hello, has there been any developments on this bug? The plugin is useless without this getting fixed. Thank you for your time.

**Comment by Kingdingbat** [comment]
I am also having this issue.
1. Click credit card logo
2. Enter credit card information into form
3. Click "Pay Now"
4. "Something went wrong..." error
5. Click "Try again", Paypal modal loads
6. Enter credit card again
7. Payment processes fine.

**Comment by bmccotter** [comment]
@EktaPayPal  any chance you can help us resolve?

**Comment by 1975bigmac1975** [comment]
I'm having exactly the same issue, any news on a fix?

**Comment by yukikatayama** [comment]
@allendav pinging for any possible updates? 

Getting more users concerned about this. Some in forums here:

https://wordpress.org/support/topic/your-plugin-is-still-broken/
https://wordpress.org/support/topic/smart-payment-buttons-broken-still/
https://wordpress.org/support/topic/the-checkout-error-something-went-wrong-happens-everytime/

**Comment by dechov** [comment]
Thank you @yukikatayama

The "You have not loaded metadata" error was fixed on PayPal's end a few weeks ago.

I have run into some other issues – including the "Something went wrong" message after completing the credit card form – and reported to PayPal but currently cannot reproduce them in production so they may be specific to the sandbox environment. Has this been seen in production, and if so is it still occurring? @fhaps @card1concepts @Kingdingbat @1975bigmac1975

I still have yet to run into the original error reported in this issue (error code 10002).

**Comment by EktaPayPal** [comment]
+ Khang to the loop as well!

Thanks and Regards,

Ekta Agarwal
Technical Account Manager, Professional Services
Mobile: 408.373.2287
Skype: paypal_ekta
[PP_h_logo_30x123]

**Comment by khanghoang** [comment]
@EktaPayPal thanks for adding me.

Folks, we've released the fix for the missing metadata issue early this month. Please double-check if you can still reproduce it on sandbox or production.

**Comment by onlysli** [comment]
Hello,
I still get the error "An error occurred while calling the PayPal API" using latest versions of the plugins.

**Comment by 1975bigmac1975** [comment]
> Thank you @yukikatayama
> 
> The "You have not loaded metadata" error was fixed on PayPal's end a few weeks ago.
> 
> I have run into some other issues – including the "Something went wrong" message after completing the credit card form – and reported to PayPal but currently cannot reproduce them in production so they may be specific to the sandbox environment. Has this been seen in production, and if so is it still occurring? @fhaps @card1concepts @Kingdingbat @1975bigmac1975
> 
> I still have yet to run into the original error reported in this issue (error code 10002).

Still happening in a live situation, it does however work from a mobile phone so I have no idea what this means.

**Comment by card1concepts** [comment]
So this issue thread seems to be all over the place. Has issue been resolved yet?

**Comment by bmccotter** [comment]
@card1concepts  we're working on getting direction from PayPal on this. 

@khanghoang  @EktaPayPal please see above ^^ any updates on your side?

**Comment by EktaPayPal** [comment]
Team,
   PayPal released the fix for the missing metadata issue already last month.

With regards to the “Something went wrong message”, this is not a production issue, it seemed to be happening only on sandbox.

Please do let us know if you are seeing otherwise?


Thanks and Regards,

Ekta Agarwal
Technical Account Manager, Professional Services
Mobile: 408.373.2287
Skype: paypal_ekta
[PP_h_logo_30x123]

**Comment by EktaPayPal** [comment]
I also wanted to update the group that the sandbox issue has been fixed as well.

Please let us know if people are still facing issues?

Thanks and Regards,

Ekta Agarwal
Technical Account Manager, Professional Services
Mobile: 408.373.2287
Skype: paypal_ekta
[PP_h_logo_30x123]



From: "Agarwal, Ekta" <eagarwal@paypal.com>
Date: Monday, October 7, 2019 at 2:25 PM
To: woocommerce/woocommerce-gateway-paypal-express-checkout <reply@reply.github.com>, woocommerce/woocommerce-gateway-paypal-express-checkout <woocommerce-gateway-paypal-express-checkout@noreply.github.com>
Cc: Mention <mention@noreply.github.com>, "Hoang Trieu, Khang" <khoangtrieu@paypal.com>
Subject: Re: [woocommerce/woocommerce-gateway-paypal-express-checkout] Credit card icons return error (#596)

Team,
   PayPal released the fix for the missing metadata issue already last month.

With regards to the “Something went wrong message”, this is not a production issue, it seemed to be happening only on sandbox.

Please do let us know if you are seeing otherwise?


Thanks and Regards,

Ekta Agarwal
Technical Account Manager, Professional Services
Mobile: 408.373.2287
Skype: paypal_ekta
[PP_h_logo_30x123]

**Comment by khanghoang** [comment]
Hi folks, sorry for the late reply. There is a couple of things I want to clarify:

- The original issue was the error message showed up immediately right after the buyers click on the credit card icons (https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issue-475360218 and https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-517099473). The root cause was we've loaded the wrong locale data (as pointed out by https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-517099473) (TL'DR: It was because of the locale the JS SDK used to init Inline Guest doesn't match with the buyer's shipping/billing country). **We addressed this issue by a fixed last month, and I couldn't reproduce it on both sandbox and production**.

- Folks also reported about the same error message showed up **after buyers submit the form** (which is entirely different and should be another issue). The reason for this error is mainly because of the wrong credit number, invalid billing information, invalid CVV number, etc... FYI we're also working on the error message to make it less generic.

cc @melindahelt

**Comment by cerecerorodrigo** [comment]
Hi there

Still having this issue whit the live account. Leaving a photo of my particular case and console log.

![image](https://user-images.githubusercontent.com/56280561/66355804-39437380-e92e-11e9-9b25-e9633a1189e7.png)

**Comment by khanghoang** [comment]
@cerecerorodrigo it's not the same with the original issue. Would you please create an issue on paypal-checkout https://github.com/paypal/paypal-checkout-components and tag me in. I'm happy to answer there.

**Comment by cerecerorodrigo** [comment]
Thank you! I'm on it.

Greetings

**Comment by allendav** [comment]
Sounds like things are now fixed on PayPal's end and this issue can be closed?

**Comment by EktaPayPal** [comment]
I believe so Allen! Please do let me know if you still hear differently!

Thanks and Regards,

Ekta Agarwal
Technical Account Manager, Professional Services
Mobile: 408.373.2287
Skype: paypal_ekta
[PP_h_logo_30x123]

**Comment by v18** [comment]
There were a few issues mentioned in this ticket. I'm going to outline them all to make sure no one's issue was missed. I will also close this ticket because the two original issues discussed here were resolved.

Thank you very much @EktaPayPal and @khanghoang for responding, fixing the problems, and reporting back 🙌

# Issue 1: Clicking credit card icons in cart and checkout resulted in an error

(Seeing a red "Error 10002" after page reload)

Reported: https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issue-475360218 by melindahelt on behalf of a user

This was reported in 2200569-zen. It was resolved by PayPal and we got confirmation that it has since been resolved.

# Issue 2: Click a credit card icon results in an inline message 

Reported originally here: https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-517099473

There were multiple reports of this:

1) Reported by @syllie here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-517099473

2) Reported by @fhaps on behalf of a user here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-518242769

3) Reported by @card1concepts here (this seems like it could be a different issue, though):
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-519490540

4) Reported by @laidlow73 here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-522594186

5) Reported by @LeRobert here (could be a different issue though):
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-523886438

6) Reported by @Kingdingbat here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-527556729

7) Reported by @1975bigmac1975  here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-529185023
@1975bigmac1975 mentioned that "this is happening in a live situation, it does however work from a mobile phone so I have no idea what this means."

The issue was reported as resolved by PayPal. Some of the issues above, however, may not be the same, and some may be error messages reporting credit card processing errors correctly (e.g. invalid CVV numbers) as mentioned here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-539231616

However, if it's still happening for anyone, please do open a new issue on this repo with a description of your issue and these details:
- the steps you took right before seeeing this error message
- a screenshot showing the error message on your screen
- a screenshot showing any javascript errors in your console

You can open a new issue here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/new/choose

# Issue 3: Inline issue by cerecerorodrigo

`Uncaught error: inline_guest_window_onerror_payment_sdk`

Reported here: https://github.com/paypal/paypal-checkout-components/issues/1224

# Issue 4: "An error occurred while calling the PayPal API"
Reported by onlysli here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-534536318

# Follow up:

## For @1975bigmac1975

For @1975bigmac1975: are you still getting an error using PayPal? If yes, could you please create a new issue here and post with these details:
- What version of WooCommerce and the PayPal plugin you are using
- a screenshot showing the error message on your screen
- a screenshot showing any javascript errors in your console

You can create a new issue here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/new/choose

# For @onlysli

For @onlysli - the error message you saw ("An error occurred while calling the PayPal API") indicates a different problem. Could you please open a new issue here with these details:
- the steps you took right before seeeing this error message
- a screenshot showing the error message on your screen
- a screenshot showing any javascript errors in your console

You can create a new issue here:
https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/new/choose

Thank you all!

**Comment by VinodSoftcripts** [comment]
Hi Everyone,

We are currently facing an issue with Woocommerce PayPal express checkout plugin,

When we are trying to do the payment with PayPal credit or debit card option is showing the following error “something went wrong. We’ll take you back to checkout so you can try again.”

Any help would be appreciated.

Thank you

**Comment by EktaPayPal** [comment]
Lauren,
   Would you happen to know if this issue is happening just for this merchant or again thru PP as a whole please?

Thanks and Regards,

Ekta Agarwal
Manager, Partner Technical Engagement
Global Professional Services

Mobile: 408.373.2287
Skype: paypal_ekta
[PP_h_logo_30x123]

**Comment by Johanarh80** [comment]
I am also presenting the same error as @cerecerorodrigo and @VinodSoftcripts, this occurs when making the payment on the checkout, mini cart and product page. Woocommerce is being used

http://prnt.sc/t1snbv
http://prnt.sc/t1sngg

Thank you

**Comment by jorgeatorres** [comment]
Hi @VinodSoftcripts @Johanarh80!

Sorry to hear you're having trouble with PayPal Checkout. In order to help us find the cause, we need to know more about your environment and what you were doing before you got the error message.
In order for us to communicate better and keep things tidy, it'd be great if you could create a [new issue here](https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/new) making sure to include the following information:

- **Environment details** such as the mode PayPal Checkout is in (sandbox or live), whether you were logged-in to PayPal before initiating the purchase (and, if so, the nature of this account: developer account, sandbox?).
- **Steps you took right before seeing the error message.**
   This includes the page you were on, which type of product(s) you had in the cart, whether you saw the message right after clicking the credit card button or after completing the credit card form, and any other info that might be relevant.
- **A screenshot showing the Javascript console errors (if any).**
   Do this after seeing the error message. Depending on your browser, you can get to the Javascript console this way: Chrome (View > Developer > Javascript Console), Firefox (Tools > Web Developer > Web Console), Safari (Develop > Show JavaScript Console).

**Comment by EktaPayPal** [comment]
This looks like an issue on PP side, and my team is looking into the same. Please stay tuned as I get back to you on the same shortly.

**Comment by naya2sangyun** [comment]
I have same issue with credit card issued in government states in US. 'something went wrong' message appears to my client and to me as well. Anybody fixed this problem? 

I've checked console message as followings:

DevTools failed to load SourceMap: Could not load content for https://www.paypal.com/smart/smart-payment-buttons.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by EktaPayPal** [comment]
Hi Team I finally have a way forward. The prescribed workaround that the engineering team has proposed is to add the upgradeLSAT parameter and set the value to true within the paypal.Buttons rendering function.Code sample:
    <script>         // Render the PayPal button into #paypal-button-container         paypal.Buttons({                     // feature flag to test fixes on order completion                     upgradeLSAT: true,            // Set up the transaction             createOrder: function(data, actions) {                 return actions.order.create({                     purchase_units: [{                         amount: {                             value: '0.01'                         }                     }]                 });             },            // Finalize the transaction             onApprove: function(data, actions) {                 return actions.order.capture().then(function(details) {                     // Show a success message to the buyer                     alert('Transaction completed by ' + details.payer.name.given_name + '!');                 });             }         }).render('#paypal-button-container');     </script>

**Comment by EktaPayPal** [comment]
Hi Team I have a way forward here


The prescribed workaround that the engineering team has proposed is to add the upgradeLSAT parameter and set the value to true within the paypal.Buttons rendering function.Code sample:
    <script>         // Render the PayPal button into #paypal-button-container         paypal.Buttons({                     // feature flag to test fixes on order completion                     upgradeLSAT: true,            // Set up the transaction             createOrder: function(data, actions) {                 return actions.order.create({                     purchase_units: [{                         amount: {                             value: '0.01'                         }                     }]                 });             },            // Finalize the transaction             onApprove: function(data, actions) {                 return actions.order.capture().then(function(details) {                     // Show a success message to the buyer                     alert('Transaction completed by ' + details.payer.name.given_name + '!');                 });             }         }).render('#paypal-button-container');     </script>

**Comment by EktaPayPal** [comment]
Hi Guys I have a way forward here. 
<img width="935" alt="Screen Shot 2020-07-10 at 12 33 07 PM" src="https://user-images.githubusercontent.com/27873242/87191573-8da6c580-c2a9-11ea-8652-f16c0c940ed9.png">

**Comment by EktaPayPal** [comment]
Please let me know if you continue to see a problem after applying the same

**Comment by sitelet** [comment]
Ekta, 
Tried out you suggestion and used upgradeLSAT:true in the paypal.rendering function. But I am still seeing the issue. Would luv to hear any other solution, if there is any?
Thanks

**Comment by davidboon78** [comment]
I'm experiencing the same issue.  This bug more or less renders woocommerce entirely useless it seems.  Here's my details.
Environment: Live, using default PayPal payment, but had the same error with the paypal express plugin, also shipping from Canada

Steps taken: Entered paypal from checkout page, clicked "Proceed to Paypal" button, from paypal.com/webapps/hermes I clicked the "pay with a credit or Visa Debit Card", filled out information, received "things don't appear to be working at the moment" error

Screenshot
[https://boomtechgear.com/wp-content/uploads/2020/08/PayPal-Checkout-Please-try-again.-Personal-Microsoft%E2%80%8B-Edge-8_29_2020-12_03_57-PM-768x419.png](https://boomtechgear.com/wp-content/uploads/2020/08/PayPal-Checkout-Please-try-again.-Personal-Microsoft%E2%80%8B-Edge-8_29_2020-12_03_57-PM-768x419.png)

**Comment by bmccotter** [comment]
@EktaPayPal any insight as to why this is still happening?

**Comment by dboon1978** [comment]
FIXED: I solved this issue by starting a new paypal account.  The paypal account I was using was converted from a standard account.

**Comment by strarsis** [comment]
@dboon1978: So a PayPal account that has been upgraded from a standard account will have this subtle (API?) issue?
So this is an issue with PayPal, that PayPal has to fix?

**Comment by davidboon78** [comment]
@strarsis  I can't speak to the exact reason for other peoples' issues, but rather than waste time trying to track down the problem,, it seems a lot easier to just make a new account and setup my banking info on the new account.

**Comment by givicha** [comment]
It is better to register a new account If this will definitely solve the problem. 
Someone else tried to do that and no longer has this problem?

**Comment by akumar8k** [comment]
I am able to do with Credit/Debit card, but only in private browser. before this, I tried with PayPal login that's why maybe not working with debit card in normal browser.

**Comment by jorgeatorres** [comment]
Hi everyone,

Please refer to comments https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-539231616 and https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-553194018 for details on what might be happening and how to address.

The credit card fields are directly rendered and handled by PayPal's scripts, which means the error is frequently the result of a validation error or just a temporary glitch. Because of this and to keep things tidy, I'm going to close this issue now but if anyone is still having trouble and can consistently reproduce the problem, please open a new issue with all the details as [described here](https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/596#issuecomment-646098768) and we'll be happy to take a look.

**Comment by Ivajkin** [comment]
The issue persists. Example: PayHip product pages.
